README.md

```diff
@@ -1,58 +1,120 @@
+    <img src="https://badge.fury.io/js/react-native.svg" alt="Current npm package version." />
+  </a>
+  <a href="https://facebook.github.io/react-native/docs/contributing">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=reactnative">
+    <img src="https://img.shields.io/twitter/follow/reactnative.svg?label=Follow%20@reactnative" alt="Follow @reactnative" />
+  </a>
+</p>
+
+<h3 align="center">
+  <a href="https://facebook.github.io/react-native/docs/getting-started">Getting Started</a>
+  <span> · </span>
+  <a href="https://facebook.github.io/react-native/docs/tutorial">Learn the Basics</a>
+  <span> · </span>
+  <a href="https://facebook.github.io/react-native/showcase.html">Showcase</a>
+  <span> · </span>
+  <a href="https://facebook.github.io/react-native/docs/contributing">Contribute</a>
+  <span> · </span>
+  <a href="https://facebook.github.io/react-native/en/help">Community</a>
+  <span> · </span>
+  <a href="https://github.com/facebook/react-native/blob/master/.github/SUPPORT">Support</a>
+</h3>
+
+React Native brings [**React**'s](https://reactjs.org/) declarative UI framework to iOS and Android. With React Native, you use native UI controls and have full access to the native platform.
+
+- **Declarative.** React makes it painless to create interactive UIs. Declarative views make your code more predictable and easier to debug.
+- **Component-Based.** Build encapsulated components that manage their own state, then compose them to make complex UIs.
+- **Developer Velocity.** See local changes in seconds. Changes to JavaScript code can be live reloaded without rebuilding the native app.
+- **Portability.** Reuse code across iOS and Android.
+
+## Contents
 
 - [Requirements](#requirements)
-- [Getting Started](#building-your-first-react-native-app)
-- [Documentation](#full-documentation)
-- [Upgrading](https://facebook.github.io/react-native/docs/upgrading)
-- [Contributing](#join-the-react-native-community)
-- [Code of Conduct](./CODE_OF_CONDUCT.md)
+- [Building your first React Native app](#building-your-first-react-native-app)
+- [Documentation](#documentation)
+- [Upgrading](#upgrading)
+- [How to Contribute](#how-to-contribute)
+- [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
----
 
-## Requirements
+## 📋 Requirements
 
-Supported target operating systems are >= Android 4.1 (API 16) and >= iOS 9.0. You may use Windows, macOS, or Linux as your development operating system, though building and running iOS apps is limited to macOS by default (tools like [Expo](https://expo.io) can be used to get around this).
+React Native apps may target iOS 9.0 and Android 4.1 (API 16) or newer. You may use Windows, macOS, or Linux as your development operating system, though building and running iOS apps is limited to macOS by default. Tools like [Expo](https://expo.io) can be used to work around this.
```

Comment:
“macOS by default” - I recommend removing the “by default” since it’s a platform limitation. The next sentence helps with the workaround.